src/content/docs/new-relic-solutions/increase-user-engagement.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Increase user engagement with website performance monitoring
+metaDescription: Capture developer-grade insights with website performance monitoring (WPM) so you can make data-informed improvements to your site. 
+---
+
+import side1wpm from 'images/tutorial_screenshot-crop1_full-wpm.webp'
+
+import side2wpm from 'images/tutorial_screenshot-crop2_full-wpm.webp'
+
+import side1Bwpm from 'images/tutorial_screenshot-crop1b_full-wpm.webp'
+
+import side2Bwpm from 'images/tutorial_screenshot-crop2b_full-wpm.webp'
+
+import pingWPM from 'images/tutorial_screenshot-crop_ping-wpm.webp'
+
+import cwvWPM from 'images/tutorial_screenshot-crop_cwv-wpm.webp'
+
+import timeWPM from 'images/tutorial_screenshot-crop_cwv-time-wpm.gif'
+
+import contentWPM from 'images/tutorial_screenshot-crop_content-wpm.webp'
+
+import stepOne from 'images/tutorial_screenshot-crop_step1-wpm.webp'
+
+import pageSpeedAPI from 'images/tutorial_screenshot-full_pageSpeed-wpm.gif'
+
+Maintaining a website combines the effort of multiple teams. When outages happen, developer and engineering teams have access to resolve the issue directly, but a digital marketer has a different situation. While your team's success is just as dependent on site performance, you might not have the tools to identify or address any issue yourselves.
+
+Our website performance monitoring (WPM) tool gives you insight about your site's availability and performance. With out of the box alerts and routine checks from our monitors, you'll never be in the dark about user experience. 
```

Comment:
```suggestion
Our website performance monitoring (WPM) tool gives you insight into your site's availability and performance. With out-of-the-box alerts and routine checks from our monitors, you'll never be in the dark about user experience. 
```

Reply:
Does out-of-the-box even translate well to other languages? It isn't in the merriam-webster dictionary and sounds like an idiom.

Reply:
"in the dark" is also an idiom.